doc/mgr/diskprediction.rst

```diff
@@ -0,0 +1,113 @@
+=====================
+DiskPrediction plugin
+=====================
+
+Disk Prediction plugin is used to collect disk information from Ceph OSD and
+send DiskPrediction server. The disk information includes the following.
```

Comment:
It's not obvious to someone reading this document what a "DiskPrediction server" is.  If it's a specific piece of software (the diskprophet product?) then this should be more specific and include a link.